### PR TITLE
Append STYLE to HTMLHeadElement

### PR DESCRIPTION
**Pull request by triplekdev**
Hi, I added a `toHEAD` function to support in-browser utility to directly append/insert the CSS and or the JSON created by the library, very simple though. 

I hope you find this useful and can be merge. If not, please Ignore.

Thanks for the library.

### CODE REVIEW

File: cssjson.js [hunk 1]
```diff
@@ -58,6 +58,10 @@ var CSSJSON = new function () {
         return typeof x == 'undefined' || x.length == 0 || x == null;
     };
 
+	var isCssJson = function (node) {
+		return !isEmpty(node) ? (node.attributes && node.children) : false;
+	}
```
Comment (aramk) [inline cssjson.js]: Could you please fix the indentation for this and the toHead() method? I used 4 spaces.

**Comment by aramk** [comment]
Thanks, just a minor comment.

File: cssjson.js [hunk 2]
```diff
@@ -208,9 +212,62 @@ var CSSJSON = new function () {
...
-    var strAttr = function (name, value, depth) {
+	var isValidStyleNode = function (node) {
+		return (node instanceof HTMLStyleElement) && node.sheet.cssRules.length > 0;
+	}
+
+	var timestamp = function () {
+		return Date.now() || +new Date();
+	};
+
+	var strAttr = function (name, value, depth) {
```
Comment (aramk) [inline cssjson.js]: I think the indentation here might also need some fixing.

**Comment by triplekdev** [comment]
@aramk Please re-check. I am using Atom w/ hard tabs, sorry about that.

**Comment by aramk** [comment]
No worries, thanks for that!

**Comment by triplekdev** [comment]
Thanks!

**Pull request merged**